tests/agent_server/deploy/test_scheduler_agent.py

```diff
@@ -180,9 +180,7 @@ async def test_basic_deploy(agent: TestAgent, make_resource_minimal):
         ResourceIdStr(rid2): make_resource_minimal(rid2, {"value": "a"}, [rid1], 5),
     }
 
-    # FIXME: SANDER: It seems we immediatly deploy if a new version arrives, we don't wait for an explicit deploy call?
-    # Is this by design?
```

Comment:
It was, with the intention to have the same behavior as the one we have now. But I just realized that it is actually a deviation since this is currently managed by a setting.

I have to give it more thought, but it may be difficult to change this, damn.

Reply:
We will have to dampen this somehow, but not now.